src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -17,18 +19,39 @@ public class FindCommandParserTest {
 
     @Test
     public void parse_emptyArg_throwsParseException() {
-        assertParseFailure(parser, "     ", String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
```

Comment:
Just a nit, but maybe it would be good to also test for when we input the correct hotkey, but not the correct arguments. Ie. just "find "

Reply:
Good catch! Just added.